test-framework/common/src/main/java/io/quarkus/test/common/TestHostLauncher.java

```diff
@@ -28,7 +28,7 @@ public void close() throws IOException {
 
     @Override
     public boolean listensOnSsl() {
-        return false;
+        return Boolean.parseBoolean(System.getProperty("quarkus.http.test-ssl-enabled", "false"));
```

Comment:
Won't this trigger a warning about unknown configuration when it's set? Or is it a known configuration already? I couldn't find it at a quick glance but I might have missed something.

From what I can see `quarkus.http.test-host` would also trigger a warning.

If so, I think we should try a bit harder.

Reply:
I expressed the same concern in https://github.com/quarkusio/quarkus/pull/34507#discussion_r1251380134

Reply:
Right, I thought we had that, but I missed it. In any case, let's not burden @pcasaes with this, I'll take care of it in a follow up

Reply:
Thanks!

I've seen those warnings and I wouldn't know where to begin frankly. My understanding (which is likely incorrect!) is that the integration test suite run as POJO with little to know involvement from the framework (ie: no inject support).